docscan/client.go

```diff
@@ -243,12 +243,17 @@ func (c *Client) DeleteMediaContent(sessionID, mediaID string) error {
 }
 
 // GetSupportedDocuments gets a slice of supported documents
-func (c *Client) GetSupportedDocuments() (*supported.DocumentsResponse, error) {
+func (c *Client) GetSupportedDocuments(isStrictlyLatin bool) (*supported.DocumentsResponse, error) {
+	includeNonLatin := "0"
```

Comment:
this logic is the inverse of what we need and can be removed by having includeNonLatin as the function's param